cli/templatepush.go

```diff
@@ -505,6 +482,44 @@ func ParseProvisionerTags(rawTags []string) (map[string]string, error) {
 	return tags, nil
 }
 
+var (
+	warnNoMatchedProvisioners = `No provisioners are available to handle the job!
+Please contact your deployment administrator for assistance.
+Details:
+	Provisioner job ID : %s
+	Requested tags     : %s
+`
+	warnNoAvailableProvisioners = `All available provisioner daemons have been silent for a while.
```

Comment:
What does "silent" mean in this context?

Does this mean there _are_ daemons available but they haven't sent a heartbeat to the control-plane in a while? Looks like the stale interval is 90s and the default heartbeat interval is 60s, so it just has to miss one heartbeat to be considered "silent".

Forgive me if this is rehashing a discussion which happened elsewhere, but what is the operational value of this warning? We already show a position in the queue when the queue is backed up. Is there another scenario besides a deep queue that we're solving for here?

Reply:
I think I lifted the copy from Sas, but there's been some drift since then. I'll update the copy.